docs/concepts.md

```diff
@@ -74,7 +74,7 @@ the configurable default is unset, DCE uses a period of seven (7) days.
 A lease that is _over budget_ has exceeded the budget amount set
 by the `budgetAmount` field of the lease.
 
-Each lease has a configurable _[budget](#budget)_. DCE periodically
+Each lease has a configurable [budget](#budget). DCE periodically
```

Comment:
Is this PR attempting to fix all the links? this one isn't rendering, and I'm still seeing a bunch of errors from Sphinx.
 Maybe we should put this effort in another PR?